understanding/20/headings-and-labels.html

```diff
@@ -93,6 +96,9 @@ <h2>Examples of Headings and Labels</h2>
          <dt>A form asking for the name of the user</dt>
          <dd>A form asks for the name of the user. It consists of two input fields to ask for the first
                and last name. The first field is labeled <q>First name</q>, the second is labeled <q>Last name</q>.</dd>
+         <dt>A search field labeled by a loupe icon</dt>
```

Comment:
probably also referring to this as a "magnifying glass"

not everyone is familiar with the "loupe" term

i'll similarly add the "loupe" term to the 3.3.2 PR

Reply:
good point, will change to magnifying lass

Reply:
thanks!   ended up doing a larger review (that i kept having to pickup / put down over the last hour) so some additional suggestions for you to consider.